modules/commons/src/main/resources/schemas/neurosciencegraph/commons/variables/v0.2.0.json

```diff
@@ -0,0 +1,136 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.2.0",
+    {
+      "@base": "{{base}}/schemas/neurosciencegraph/commons/variables/v0.2.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "shapes": [
+    {
+      "@id": "AlgebraicValueShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "property": [
+        {
+          "path": "schema:unitCode",
+          "name": "Unit code",
+          "datatype": "xsd:string",
+          "minCount": 1,
+          "maxCount": 1
+        },
+        {
+          "path": "nsg:statistic",
+          "name": "Statistic",
+          "description": "The name of the statistic describing the sample.",
+          "datatype": "xsd:string",
+          "maxCount": 1
+        }
```

Comment:
Alternatively, AlgebraicValueShape could be modelled like the AgeShape in the subject schema:

```   
 {
      "@id": "this:AgeShape",
      "@type": "sh:NodeShape",
      "property": [
        {
          "path": "nsg:period",
          "name": "Period",
          "in": [
            "Pre-natal",
            "Post-natal"
          ],
          "minCount": 1,
          "maxCount": 1
        },
        {
          "path": "schema:value",
          "name": "Age value",
          "node": "{{base}}/schemas/neurosciencegraph/commons/quantitativevalue/v0.1.0/shapes/QuantitativeValueShape",
          "minCount": 1,
          "maxCount": 1
        }
      ]
    }
```
by extending the QuantitativeValueShape with an nsg:statistic property shape